hitmap.go

```diff
@@ -49,6 +50,7 @@ func (hits *hitMap) expireLimits() {
 	for {
 		hitMapCopy := hits.getMap()
 		for _, ipr := range hitMapCopy {
+			// TODO address concurrent read
```

Comment:
`ipr.limited` is being read in the condition. I have been told that *may* be an issue?